homeassistant/components/ffmpeg/camera.py

```diff
@@ -43,14 +51,14 @@ class FFmpegCamera(Camera):
 
     _attr_supported_features = CameraEntityFeature.STREAM
 
-    def __init__(self, hass, config):
+    def __init__(self, hass: HomeAssistant, config: dict[str, Any]) -> None:
         """Initialize a FFmpeg camera."""
         super().__init__()
 
-        self._manager = hass.data[DATA_FFMPEG]
-        self._name = config.get(CONF_NAME)
-        self._input = config.get(CONF_INPUT)
-        self._extra_arguments = config.get(CONF_EXTRA_ARGUMENTS)
+        self._manager: FFmpegManager = hass.data[DATA_FFMPEG]
+        self._name: str = config[CONF_NAME]
+        self._input: str = config[CONF_INPUT]
+        self._extra_arguments: str = config[CONF_EXTRA_ARGUMENTS]
```

Comment:
Looking at camera.py, aren't these config entries optional?

Are you sure it is ok to remove the get()?

Reply:
They are either required or have a default value. So it should be fine.
https://github.com/home-assistant/core/blob/76a1e979476822ac2abef168cb60b7dddb86b291/homeassistant/components/ffmpeg/camera.py#L22-L28